### ISSUE TITLE
[vite] error while updating dependencies: Error: ENOENT: no such file or directory, open 'C:/Users/onebuf/Desktop/website/node_modules/.store/vue-axios@3.5.2/node_modules/vue-axios/dist/vue-axios.esm.min.js'

### ISSUE THREAD

**Issue by ONEBUF** [axios/axios#5479]
### Describe the bug

![image](https://user-images.githubusercontent.com/24307511/213533854-40cb191d-28ea-45ce-95db-d379f421442a.png)

I  JUST DONT KNOW WHAT HAPPEN

### To Reproduce

just after install default-passive-events

**Comment by China-li-bai** [comment]
Hotfix
If you ever get this error, the hotfix is to follow these steps:

delete node-modules folder
run command npm cache clean --force
run command npm install
install the package again with npm install your-package-name
It should all work fine after that.

If these commands still don’t solve your issue, and you use git as a source control system, you can try the unbeatable git clean -fdx command. After that, run npm install again. Beware – it removes all files not checked in to git. It may also remove your IDE settings etc., so use with care.

But it’s only a hotfix, a solution for now to unblock you.

**Comment by China-li-bai** [comment]
修复文件路径
作为上一个解决方案的扩展，您还可以尝试手动修复错误报告丢失的文件路径。执行此操作的具体方法因框架而异，因此您必须查找正在使用的特定框架的文档，以检查如何修改文件路径。

如果确定配置文件或依赖项存在，则可以手动将 Web 服务器指向正确的文件。

重新初始化项目
您可能还必须重新初始化项目以刷新正在使用的所有依赖项。这可以通过删除node_modules文件夹和 package-lock.json 文件来完成。只需打开一个终端，转到项目的根文件夹，然后一次运行一个命令。

rm -rf node_modules package-lock.json
npm install

**Comment by jasonsaayman** [comment]
i think this is probably an issue with vue-axios, if this is still an problem please open a new issue with a reproducible example and mention me
